src/bashly.yml

```diff
@@ -100,21 +107,19 @@ commands:
   - long: --v6d-version
     arg: v6d-version
     help: v6d version to clone
-    default: v0.11.3
+    default: v0.11.6
```

Comment:
is that possible for sth. like `latest` as default?

Reply:
Could use main as latest.